tdrs-backend/tdpservice/users/test/test_auth.py

```diff
@@ -37,6 +39,34 @@ def json(self):
         return self.data
 
 
+@pytest.fixture
+def patch_login_gov_jwt_key(settings):
+    """Override JWT Key setting with the key needed for tests."""
+    assert test_private_key is not None, 'Missing env var: JWT_CERT_TEST'
+    settings.LOGIN_GOV_JWT_KEY = test_private_key
+
+
+@pytest.fixture
+def mock_token():
+    """Retrieve the mock token to be used for tests."""
```

Comment:
This default was previously defined in settings, but there is no use for this value to be exposed as a setting as this test module is the only reference.